shark/shark_importer.py

```diff
@@ -122,3 +122,15 @@ def forward(self, inputs=None):
             print("Inference", self.model_source_hub, " not implemented yet")
         elif self.model_source_hub == "jaxhub":
             print("Inference", self.model_source_hub, " not implemented yet")
+
+
+def shark_load(model_name, file_path):
```

Comment:
The name should be something that specifies loading from shark_hub like `load_from_shark_hub()` or `shark_hub_load()`.

Reply:
I like shark_load() to keep it small and simple but I am not sure if we want to call hub / zoo yet (maybe tank ?) but this is a marketing exercise :D